### ISSUE TITLE
Better description for lbd1 in do.disr

### ISSUE THREAD

**Issue by antoine-sachet** [kyoustat/Rdimtools#12]
First of all, a big thank you for this amazing package! 

I have been researching Diversity-Induced Self-Representation (DISR) FS and I think the description for lbd1 and lbd2 in DISR could be clearer.

It currently says:

```
#' @param lbd1 nonnegative number to control the degree of self-representation.
#' @param lbd2 nonnegative number to control the degree of feature similarity.
```

which suggests that the function being minimised is basically `lbd1 * [term for self-rep error] + lbd2 * [term for similarity]`.

But reading the paper (and the Rcpp code to check it matches), it looks like `lbd1` controls the regularisation of the regularised linear model (used to estimate self-representation) and  `lbd2` is the weight of the similarity term in the objective function (used to quantify diversity). 

Basically the objective function actually has the form `[linear model error + lbd1 * regularisation term] + lbd2 * [term for similarity]`, so `lbd1` doesn't control the degree of self-representation -- that's all in `lbd2`, which controls the trade-off between self-rep and diversity.

I think a more accurate description would be:

```
#' @param lbd1 nonnegative number to control the degree of regularisation of the self-representation.
#' @param lbd2 nonnegative number to control the degree of feature diversity. `lbd2=1` gives equal weight to self-representation and diversity.
```

**Comment by kisungyou** [comment]
thanks @antoine-sachet , your suggestion will be reflected in the following version.
